File: C#/TollCalculator/TollCalculator/Services/TollFreeVehicles.cs
using System.Collections.Generic;
using TollFeeCalculator.Interfaces;

namespace TollFeeCalculator.Services
{
    public class TollFreeVehicles : ITollFreeVehicles
    {
        private readonly List<VehicleType> _tollFreeVehicles = new List<VehicleType>
        {
            VehicleType.Motorbike,
            VehicleType.Tractor,
            VehicleType.Emergency,
            VehicleType.Diplomat,
            VehicleType.Foreign,
            VehicleType.Military
        };

        public bool IsTollFreeVehicle(IVehicle vehicle)
        {
            if (vehicle == null) return false;

            return _tollFreeVehicles.Contains(vehicle.GetVehicleType());
        }
    }
}

File: toll_free_calc_app/src/main/java/com/evolve_technology/calculator/constant/TollFreeVehicles.java
package com.evolve_technology.calculator.constant;

public enum TollFreeVehicles {
	
		    MOTORBIKE("Motorbike"),
		    TRACTOR("Tractor"),
		    EMERGENCY("Emergency"),
		    DIPLOMAT("Diplomat"),
		    FOREIGN("Foreign"),
		    MILITARY("Military");
		    private final String type;

		    TollFreeVehicles(String type) {
		      this.type = type;
		    }

		    public String getType() {
		      return type;
		    }
}
